vmm/src/memory_manager.rs

```diff
@@ -716,6 +767,43 @@ impl MemoryManager {
                 .unwrap();
         }
 
+        // Apply NUMA policy if needed.
+        if let Some(nodes) = &mut host_numa_nodes {
+            let addr = region.deref().as_ptr();
+            let len = region.deref().size() as u64;
+            let mode = MPOL_BIND;
+            let mut nodemask: Vec<u64> = Vec::new();
+            let maxnode = (nodes.len() + 1) as u64;
+            let flags = MPOL_MF_STRICT | MPOL_MF_MOVE;
+
+            // Re-order the list of NUMA nodes.
+            nodes.sort();
+
+            // Now that we know the list of nodes is properly ordered, we can
+            // create a nodemask vector with the right size. The last node is
+            // the biggest one, therefore we can determine the number of u64
+            // items we need.
+            let last_node = nodes[nodes.len() - 1] as usize;
```

Comment:
This is `nodes.last()` ?

Reply:
Yes well I didn't know `last()` was available :p